### ISSUE TITLE
`check` does not seem to work on 0.4.0

### ISSUE THREAD

**Issue by mdwn** [chasinglogic/licensure#36]
Against my repo: https://github.com/mdwn/mtrack

Using `licensure 0.3.2`:

```
➜  mtrack git:(release-v0.1.3) licensure --check -p
➜  mtrack git:(release-v0.1.3) licensure -V
licensure 0.3.2
```

Using `licensure 0.4.0`:

```
➜  mtrack git:(release-v0.1.3) cargo install licensure
Replaced package `licensure v0.3.2` with `licensure v0.4.0` (executable `licensure`)
➜  mtrack git:(release-v0.1.3) licensure --check -p   
The following files were not licensed with the given config.
src/audio.rs
src/audio/cpal.rs
src/audio/mock.rs
src/config.rs
src/config/controller.rs
src/config/midi.rs
src/config/player.rs
src/config/playlist.rs
src/config/song.rs
src/config/track.rs
src/config/trackmappings.rs
src/controller.rs
src/controller/keyboard.rs
src/controller/midi.rs
src/main.rs
src/midi.rs
src/midi/midir.rs
src/midi/mock.rs
src/player.rs
src/playlist.rs
src/playsync.rs
src/songs.rs
src/test.rs
```

I've attempted to update the licenses, but there's no diff:

```
➜  mtrack git:(release-v0.1.3) licensure -ip
➜  mtrack git:(release-v0.1.3) git status
On branch release-v0.1.3
Your branch is up to date with 'origin/release-v0.1.3'.

nothing to commit, working tree clean
```

**Comment by hcldan** [comment]
I am also seeing this.

**Comment by mdwn** [comment]
I believe this is related to this PR: https://github.com/chasinglogic/licensure/pull/35, in particular this change here: https://github.com/chasinglogic/licensure/blob/master/src/licensure.rs#L51. The call to `add_license_header` returns `None` if the license is already present, which inadvertently indicates that it needs to be updated. cc @chasinglogic

**Comment by chasinglogic** [comment]
Fix in PR here: https://github.com/chasinglogic/licensure/pull/37

Once CI passes I'll get that merged and fix the linux build

**Comment by chasinglogic** [comment]
Hey all,

I've merged and released this fix to crates and github as `0.4.1`. I've tested it myself but would appreciate if you could give it a whirl @mdwn and if your issue is fixed close the ticket.
